templates/cluster-template-ext-infra.yaml

```diff
@@ -79,7 +79,6 @@ spec:
       namespace: "${NAMESPACE}"
   kubeadmConfigSpec:
     clusterConfiguration:
-      imageRepository: ${IMAGE_REPO}
```

Comment:
where do images come from if we don't supply this env var? And how would we change where the image comes from if we needed to?

Reply:
i think the answer here is that someone would have to set the `imageRepository` in their own template and not use the ones we ship